src/index.js

```diff
@@ -57,7 +57,11 @@ async function updatePackageJSON(pkg) {
     }
   }
 
-  const flowConfigs = await globby(['**/.flowconfig', '!**/node_modules/**']);
+  const flowConfigs = await globby(['.flowconfig', '!**/node_modules/**'], {
```

Comment:
Is it necessary to change from `**/.flowconfig` to `.flowconfig`?

Reply:
In the react repo it found the .flowconfig in the root from every package and added flow-type dependency in every package.json even though there is only one .flowconfig in the root.
But as I said I don't have experience with flow yet, maybe this is what should happen?

Reply:
Flow takes the closest `.flowconfig` file it can process (by traversing up the tree) and usually this config is in the root of the project. Also, Flow doesn't traverse for configs down the tree (and thus doesn't merge existing configs). Provided that the command will be run from project's root, this change is necessary.

Reply:
Now I am not sure anymore @thymikee 

What does this line? It looks for a `.flowconfig` to figure out if this is a flow project. Then later on based on this check it adds the `@babel/preset-flow` to the `package.json` in https://github.com/babel/babel-upgrade/blob/master/src/upgradeDeps.js#L57-L61
This is executed for each `package.json` it finds, so for each package in a monorepo.
Before my change it will find a `.flowconfig` in the root and add the dependency to each package. With my change it will only add the dependency to the root `package.json` and each package that brings it own `.flowconfig`.

What I do not know: Do the packages in the monorepo need this dependency or is it enough to have this in the root `package.json`?

Reply:
>With my change it will only add the dependency to the root package.json and each package that brings it own .flowconfig.

That looks like a proper behavior to me. Usually (but maybe not always), projects in a monorepo use Flow from root and don't contain their own `.flowconfig`. If some of them do, that would imply that Flow should be run directly from these projects instead from the root.

I'd say – if you find `.flowconfig` in any package in the monorepo, that package should have its own `@babel/preset-flow`.

Reply:
Thanks for clarifying this! Now I feel more confident with my changes.